apps/etherpad/templates/services/app/secret_keys.yml.j2

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: etherpad
+    service: etherpad
+  name: "{{ etherpad_file_secret_name }}"
+  namespace: "{{ project_name }}"
+data:
+  SESSIONKEY.txt: "{{ ETHERPAD_VAULT.SESSIONKEY | default('thisisasessionkey') | b64encode }}"
```

Comment:
Should we fail instead of defining a default session key ?

Reply:
@sampaccoud asked me a similar question. IMO this should be a global policy for all vault-stored values.

Reply:
But I can change this here if you insist a bit :sweat_smile:

Reply:
If you change this here, it would show us the way to a better future.

Reply:
Well said dude!